src/QsCompiler/LlvmBindings/DebugInfo/DILocation.cs

```diff
@@ -15,24 +15,22 @@ public class DILocation
     {
         /// <summary>Initializes a new instance of the <see cref="DILocation"/> class.</summary>
         /// <param name="context">Context that owns this location</param>
-        /// <param name="line">line number for the location</param>
-        /// <param name="column">Column number for the location</param>
+        /// <param name="position">1-based line/col number for the location</param>
         /// <param name="scope">Containing scope for the location</param>
-        public DILocation(Context context, uint line, uint column, DILocalScope scope)
-            : this(context, line, column, scope, null)
+        public DILocation(Context context, DebugPosition position, DILocalScope scope)
```

Comment:
Before this goes to main, I should probably change the DebugPosition back to the line, column format since this is a public API. I can move DebugPosition information to within the QIR Generation only.

Reply:
Can you please add a TODO comment about this in the code so we don't forget?